temp/style.css

```diff
@@ -1,51 +1,51 @@
 section {

```

Comment:
the files in the temp folder were temporary and simply used to store unused/test code

Reply:
Yeah i saw that earlier while I was changing the temp/style.css and nothing was changing so yeah.